assets/bower_components/three-js/three-js.html

```diff
@@ -54,7 +46,8 @@
           });
           Polymer.dom(this.root).appendChild(renderer.domElement);
           this.renderer = renderer;
-          renderer.setSize(this.w, this.h);
+          var parent = Polymer.dom(this).parentNode;
```

Comment:
> setting things up so that there is always a div

In ThreeJS.jl, we could have a div all the time. But for cases in which you want to just write html (without Julia), it's nicer to be able just write `<three-js></three-js>` and have things working I feel.